<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alt::TransactionWizard</class>
 <widget class="QDialog" name="alt::TransactionWizard">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ALT Components - Transaction</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="alt::RequestWizardPage" name="requestPage"/>
     <widget class="alt::ResolveWizardPage" name="resolvePage"/>
     <widget class="alt::ProgressWizardPage" name="progressPage"/>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="alt::ApplyStatusBar" name="statusBar" native="true"/>
     </item>
     <item>
      <spacer name="applyHorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>503</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>&lt; &amp;Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="text">
        <string>&amp;Next &gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>&amp;Apply</string>
       </property>
       <property name="icon">
        <iconset theme="emblem-default"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="finishButton">
       <property name="text">
        <string>&amp;Finish</string>
       </property>
       <property name="icon">
        <iconset theme="emblem-default"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="retryButton">
       <property name="text">
        <string>&amp;Retry</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::ProcessStop"/>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>alt::ApplyStatusBar</class>
   <extends>QWidget</extends>
   <header>ui/statusbar/applystatusbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>alt::ProgressWizardPage</class>
   <extends>QWidget</extends>
   <header>ui/applydialog/progresswizardpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>alt::RequestWizardPage</class>
   <extends>QWidget</extends>
   <header>ui/applydialog/requestwizardpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>alt::ResolveWizardPage</class>
   <extends>QWidget</extends>
   <header>ui/applydialog/resolvewizardpage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>finishButton</sender>
   <signal>clicked()</signal>
   <receiver>alt::TransactionWizard</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>649</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>alt::TransactionWizard</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
